### PR TITLE
Update to Latest version of StateChannels

### PR DESCRIPTION
**Pull request by lalexgap**
# Description
Updates the `graph-payment` to depend on the latest version of various State Channel packages.

This is based on  @geoknee 's work in #6 and should supersede #6

# How Has This Been Tested?

Relying on existing tests

# Checklist:

## Code quality
- [ ] I have written clear commit messages
- [x] I have performed a self-review of my own code
- [x] I have scoped this change as narrowly as possible
- [x] I have separated logic changes from refactor changes (formatting, renames, etc.)
- [x] I have commented my code wherever necessary
- [ ] I have added tests that prove my fix is effective or that my feature works
## Project management
- [x] I have applied the [appropriate labels](https://github.com/statechannels/statechannels/issues/3177)
- [x] I have linked to relevant issues
- [x] I have added dependent tickets
- [x] I have assigned myself to this PR
- [x] I have chosen the appropriate pipeline

### CODE REVIEW

File: packages/e2e-testing/__tests__/e2e.test.ts [hunk 1]
```diff
@@ -46,41 +46,43 @@ import {Contract} from 'ethers';
...
+      : defaultTestConfig().networkConfiguration.chainNetworkID
+  },
+  chainServiceConfiguration: {
+    attachChainService: useChain,
+    provider: process.env.RPC_ENDPOINT,
+    pk: ETHERLIME_ACCOUNTS[0].privateKey
+  }
+});
+const receiverConfig = defaultTestConfig({
+  databaseConfiguration: {connection: {database: RECEIPT_SERVER_DB_NAME}},
```
Comment (lalexgap) [inline packages/e2e-testing/__tests__/e2e.test.ts:63]: This could probably be one config

File: packages/e2e-testing/src/receipt-server.ts [hunk 2]
```diff
@@ -56,26 +57,29 @@ const commands = {
...
+          user: args.pgUsername,
+          host: 'localhost'
+        }),
+        chainServiceConfiguration: {
+          attachChainService: !!process.env.RPC_ENDPOINT,
+          provider: process.env.RPC_ENDPOINT,
+          pk: ETHERLIME_ACCOUNTS[1].privateKey
+        }
+      };
+      await DBAdmin.migrateDatabase(config);
```
Comment (lalexgap) [inline packages/e2e-testing/src/receipt-server.ts:72]: This shouldn't be necessary

Comment (lalexgap) [reply packages/e2e-testing/src/receipt-server.ts]: 48aec47

File: packages/e2e-testing/__tests__/e2e.test.ts (hunk 1 above)
Comment (lalexgap) [reply packages/e2e-testing/__tests__/e2e.test.ts]: 0b8ecee

**Review by geoknee — APPROVED** [review]

**Pull request merged**